packages/client/machines/appMusicPlaylistsMachine.ts

```diff
@@ -552,8 +552,7 @@ export function createAppMusicPlaylistsMachine({
                                             to: 'socketConnection',
                                         },
                                     ),
-
-                                    appMusicPlaylistsModel.actions.redirectToMpeLibrary(),
+                                    `redirectToMpeLibrary`,
```

Comment:
still redirecting to mpeLibrary after send req to the back to avoid rendering a white mpe room creation form confirmation screen